src/Sylius/Bundle/CoreBundle/Fixture/Factory/TaxonExampleFactory.php

```diff
@@ -73,14 +79,13 @@ public function __construct(
                 ->setDefault('description', function (Options $options) {
                     return $this->faker->paragraph;
                 })
-                ->setDefault('parent', LazyOption::randomOneOrNull($taxonRepository, 70))
-                ->setAllowedTypes('parent', ['null', 'string', TaxonInterface::class])
-                ->setNormalizer('parent', function (Options $options, $previousValue) use ($taxonManager) {
+                ->setDefault('children', [])
+                ->setAllowedTypes('children', ['array'])
+                ->setNormalizer('children', function (Options $options, $previousValue) use ($taxonManager) {
                     $taxonManager->flush();
```

Comment:
Without usage of another normalizer, this can be put somewhere inside `loadResource` method to make options resolver simpler.